.buildkite/post-commit.yml

```
@@ -30,55 +23,14 @@ steps:
     agents:
       system: x86_64-linux
 
-  - label: 'Build cardano-wallet package for Linux'
-    depends_on: nix
-    key: build-linux
-    command: nix build -o result/linux .#ci.artifacts.linux64.release
-    artifact_paths: [ "./result/linux/**" ]
-    agents:
-      system: x86_64-linux
-
-  - label: 'Build all tests (Linux)'
-    depends_on: nix
-    command: 'nix build .#ci.tests.all'
-    agents:
-      system: x86_64-linux
-
-  - label: 'Run unit tests on linux'
+  - label: 'Build all tests and run unit (Linux)'
     depends_on: nix
-    command: 'nix build -L .#ci.x86_64-linux.tests.run.unit'
+    command: 
+      - 'nix build .#ci.tests.all'
```

Should we also include nix build .#ci.benchmarks.all?

```suggestion
      - 'nix build .#ci.tests.all'
      - 'nix build .#ci.benchmarks.all'
```

---

yes, all code should buildable, before other artifacts, I'm including this